src/app/plan/components/Wizard/WizardContainer.tsx

```diff
@@ -97,6 +129,7 @@ const WizardContainer = withFormik<IOtherProps, IFormValues>({
       // TODO need to look into this closer, but it was resetting form values after pv discovery is run & messing with the UI state
       // See https://github.com/konveyor/mig-ui/issues/797
       // values.persistentVolumes = editPlanObj.spec.persistentVolumes || [];
+      values.hookDefinitions = editPlanObj.spec.hookDefinitions || [];
```

Comment:
Ah, we should probably add `hookDefinitions` to the `IPlan` interface here: https://github.com/konveyor/mig-ui/blob/master/src/app/plan/components/Wizard/types.ts#L35-L49 and then reuse the new HookDefinition interface wherever we use these values

Reply:
Removed hookDefinitions as it was unused.